universal-application-tool-0.0.1/app/controllers/FileController.java

```diff
@@ -39,6 +39,10 @@ public FileController(
     return checkApplicantAuthorization(profileUtils, request, applicantId)
         .thenApplyAsync(
             v -> {
+              // Ensure the file being accessed indeed belongs to the applicant.
```

Comment:
Suggestion (non-blocking): link to the code where this is set on create

Reply:
Done.